mason/docs/manual/manual.tex

```diff
@@ -13932,14 +13934,38 @@ \subsubsection{Building from Scratch}
     \item OpenMPI has lately been switching to version 5.0, which is not stable and also not backwards compatible with previous versions.  Well grab v4.0.5 instead:
     
     \lstinline[style=Bash]{git checkout tags/v4.0.5}
-
-    \item Alternatively, you can get the source here:\quad \lstinline[style=Bash]{https://www.open-mpi.org/software/ompi/v4.0/}
     
     \item \lstinline[style=Bash]{./autogen.pl}
     
+    \item Alternatively, you can get the source here:\quad \lstinline[style=Bash]{https://www.open-mpi.org/software/ompi/v4.0/}
+
     \item Choose a configuration option \begin{enumerate}
         \item Do this: \lstinline[style=Bash]{./configure --enable-mpi-java}\qquad (this will take quite a while)
         
+        \item An example configure on linux: 
```

Comment:
This is derived from the build file for my distributions mirror, internally this is how my system built openmpi, but I've added the java bindings